README.md

```diff
@@ -59,18 +59,42 @@ Refer to the `helper-functions.ts` file in this repository for a few common func
 
 ## API
 
-We will document the functionality provided by `graph-ts` soon. Until
-then, please check the types and modules exported from the
-[library source code](index.ts).
+Documentation on the API can be found [here](https://thegraph.com/docs/assemblyscript-api#api-reference).
 
-Also check out subgraphs that use the `graph-ts` library:
+For examples of `graph-ts` in use take a look at one of the following subgraphs:
 * https://github.com/graphprotocol/ens-subgraph
 * https://github.com/graphprotocol/decentraland-subgraph
 * https://github.com/graphprotocol/adchain-subgraph
 * https://github.com/graphprotocol/0x-subgraph
 * https://github.com/graphprotocol/aragon-subgraph
 * https://github.com/graphprotocol/dharma-subgraph  
 
+Parsing and converting data using `graph-ts`:
```

Comment:
Let's move this to the docs instead. It would be awkward to have some API documentation material here and some in the docs.